c_historical_RS_data_collation/src/combine_metadata_with_pulls.R

```diff
@@ -67,22 +67,22 @@ combine_metadata_with_pulls <- function(file_prefix, version_identifier, collati
       left_join(., metadata_light) %>% 
       mutate(DSWE = str_sub(source, -28, -24))
     # break out the DSWE 1 data
-    if (nrow(points %>% filter(DSWE == 'DSWE1') > 0)) {
+    if (nrow(points %>% filter(DSWE == 'DSWE1')) > 0) {
```

Comment:
nice parentheses catches!

Reply:
this was after I collated a bunch of data and was like... why is there no data?!